packages/cli-tool/src/commands/generate/index.ts

```diff
@@ -1,10 +1,12 @@
+import * as fs from 'node:fs';
```

Comment:
How about this, or there is something diff between these.
```suggestion
import * as fs from 'fs';
```

Reply:
4ea518e :)  (still testing...)

Reply:
Tests: All good ✅